openapi/aro_hcp/v1alpha1/openapi.json

```diff
@@ -2058,6 +2058,13 @@
             "description": "The URL of the API server of the cluster.",
             "type": "string"
           },
+          "allowed_cidr_blocks": {
+            "description": "The CIDR blocks that are whitelisted for the API server. Currently, only supported for ARO-HCP.",
```

Comment:
If this is whitelisted, does it mean that by default no access is given at all? I think this lacks definition.
Looking at the strat https://issues.redhat.com/browse/XCMSTRAT-517 it mentions:
```
    Allow a user to define allow/block lists for IP addresses or IP CIDR ranges to the API server at install time. 
    Allow updating the list of IP addresses at a later date
    Allow disabling of this feature and resort to defaults at a later date
```

Are both allow and deny lists requested? what are the requirements here?
I think whether this is the way to introduce it is not fully clear

Reply:
Understood